src/3_api.js

```diff
@@ -292,7 +292,12 @@
 		callback(new Error(utils.messages.timeout), null, 504);
 	};
 	req.onerror = function(e) {
-		callback(new Error(e.error || ('Error in API: ' + req.status)), null, req.status);
+		var url = req.responseURL || 'Unknown';
```

Comment:
Is unknown url ever possible? Would this happen then there's no error connection, or some other client error?